### PR TITLE
Fix normalize path issue on Windows

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/243
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested using internal Jenkins builds (links sent to reviewer on slack).

File: stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java [hunk 1]
```diff
@@ -79,6 +79,10 @@ else if(this.fileName.startsWith("./..")) {
 				String normalizedPath = p1.normalize().toString();
 				if (normalizedPath.length() > 0) {
 					this.fileName = normalizedPath; 
+					if (PlatformFinder.isWindows()) {
```
Comment (sophia-guo) [inline stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java:82]: I'm confused as https://github.com/AdoptOpenJDK/stf/blob/91a16e371f0ce95f3d4b7d9f769368c17ea98a67/stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java#L40 has already done the change? Why it is needed to change again?

**Review by lumpfish — APPROVED** [review]
LGTM

**Review by sophia-guo — APPROVED** [review]

File: stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java (hunk 1 above)
Comment (Mesbah-Alam) [reply stf.core/src/stf.core/net/adoptopenjdk/stf/environment/DirectoryRef.java:82]: the call to normalize() , i.e., String normalizedPath = p1.normalize().toString(); in DirectoryRef is messing changing path separators updated earlier back to Windows style ones. So, that's why the test was failing.

**Pull request merged**